### PR TITLE
Update time on refresh

### LINKED ISSUE #81: Refresh on sync not available after upgrade to Grafana 7.0.1
Before 7.x it was possible to update the clock only on dashboard refresh. It is even still mentioned in your README.md

Since upgrading to Grafana 7.0.1, our clocks are showing current time, instead of latest refresh time.

UPDATE: Tried to downgrade the plugin to 1.0.3 together with Grafana 7.0.1, but the clock does not sync correctly then either, even if the sync option appears.

### PR DESCRIPTION
**Pull request by ryantxu**
fixes #81

this was lost in migration

### CODE REVIEW

**Comment by ryantxu** [comment]
I removed the circle and codecoverage webhooks -- we can now use:
https://drone.grafana.net/grafana/clock-panel

**Review by mckn — APPROVED** [review]
LGTM!

File: src/ClockPanel.tsx [hunk 1]
```diff
@@ -19,18 +20,57 @@ export function getTimeZoneNames(): string[] {
 }
 
 class UnthemedClockPanel extends PureComponent<Props, State> {
-  timerID?: any;
+  timerID?: any = 0;
   state = { now: this.getTZ(), timezone: '' };
+  subscription?: Unsubscribable;
 
   componentDidMount() {
+    this.subscription = this.props.eventBus.subscribe({ type: 'refresh' } as any, this.onPanelRefresh);
```
Comment (torkelo) [inline src/ClockPanel.tsx:28]: this should not really be needed, if the props.data has changed there has been a refresh

Comment (ryantxu) [reply src/ClockPanel.tsx:28]: There is no query, so the update never happens

Comment (mckn) [reply src/ClockPanel.tsx]: Took @torkelo suggestion for a spin and it seems to work like a charm. So I have changed the code accordingly.

File: src/ColorEditor.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          console.log('CLICK');
+        }}
+        prefix={prefix}
+        suffix={suffix}
+      />
+    </div>
+  );
+}
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
```
Comment (torkelo) [inline src/ColorEditor.tsx:43]: any reason for using old theme here?

**Review by torkelo — APPROVED** [review]
Looks good!

File: src/ColorEditor.tsx (hunk 2 above)
Comment (mckn) [reply src/ColorEditor.tsx:43]: No, not really. I think these parts have not yet been migrated.

Comment (mckn) [reply src/ColorEditor.tsx:43]: I just moved that component from the `module.ts` into its own file.

Comment (mckn) [reply src/ColorEditor.tsx:43]: So we can probably open a separate PR to migrate this plugin to the new theme.

Comment (torkelo) [reply src/ColorEditor.tsx:43]: ah, thought it was new. no worries then

**Comment by mckn** [comment]
> I removed the circle and codecoverage webhooks -- we can now use:
> https://drone.grafana.net/grafana/clock-panel

How do we do the actual release of the plugin?

**Comment by torkelo** [comment]
> How do we do the actual release of the plugin?

no idea :( ask @briangann or @marcusolsson

**Comment by mckn** [comment]
> > How do we do the actual release of the plugin?
> 
> no idea :( ask @briangann or @marcusolsson

With the github actions that was kind of easy. But since those are removed I'm a bit in the dark here 🙈

**Comment by zuchka** [comment]
I think this is how @marcusolsson publishes plugins? That's what the recently updated docs say here:

https://github.com/grafana/plugin-catalog-review#step-9-publish-plugin-submission

to use `uploader.sh` [the `plugin-signer` repo](https://github.com/grafana/plugin-signer). Still never published a plugin so not 100% sure on the newest workflow...

**Pull request merged**